Legion/src/Assets/qc.cpp

```diff
@@ -2,142 +2,872 @@
 #include "RpakLib.h"
 #include "Path.h"
 #include "Directory.h"
+#include <version.h>
+#include <MathHelper.h>
 
-void RpakLib::ExportQC(int assetVersion, const string& Path, const string& modelPath, char* rmdlBuf, char* phyBuf)
+const std::vector<std::string> MaterialTypes = { "_rgdu", "_rgdp", "_rgdc", "_sknu", "_sknp", "_sknc", "_wldu", "_wldc", "_ptcu", "_ptcs" };
+
+#define CalculateLimit(r) ((r / Math::MathHelper::PI) * 180.0f)
```

Comment:
This macro's name is misleading. This calculation converts angles from Radians to Degrees